Documentation/spelling_wordlist.txt

```diff
@@ -330,6 +330,7 @@ decrypted
 deepcopy
 deletetopic
 demux
+dependant
```

Comment:
It's spelled "dependent" :-)

Reply:
Depends on where you are from I suppose. I don't know if we have a US vs UK spelling policy